SiteSurvey/scripts/capture.sh

```diff
@@ -55,6 +55,8 @@ if [ "$1" = "start" ]; then
 
   echo ${BSSID} > ${LOCK}
 
+  # make sure the folder exists
+  mkdir -p /pineapple/modules/SiteSurvey/capture
```

Comment:
Do you think it would be a good idea to add this line in the dependencies.sh script? and have this line check and create the directory if it does not already exist?

Also I think you can drop the -p flag since these scripts are already executing from within one level lower than the directory were creating.

Reply:
dependencies.sh is probably the better place for this if we dont want to add an empty dir to the repo. I was being lazy and using -p to prevent errors if the directory already existed. I have pushed up some new changes

Reply:
Looks good!